### PR TITLE
Add -no-bold flag to disable bold results

### PR DESCRIPTION
**Pull request by carlosgarciafm**
This is response to the issue #5.

### CODE REVIEW

**Comment by svenstaro** [comment]
Cool! Please also put that command in the README because otherwise we really don't have a lot of command discoverability.

File: src/calc.c [hunk 1]
```diff
@@ -64,6 +64,12 @@ typedef struct
 #define CALC_COMMAND_OPTION "-calc-command"
 
 
+/**
+ * Flag to disable bold results
+ */
+#define RESULT_BOLD_OPTION "-no-bold"
```
Comment (svenstaro) [inline src/calc.c]: It might make more sense to call this `NO_BOLD_OPTION` to be consistent and true to its effect.

Comment (carlosgarciafm) [reply src/calc.c]: You're right, your suggestion makes more sense.

**Comment by svenstaro** [comment]
Sweet! Good work.

**Pull request merged**